tests/test_to_container.py

```diff
@@ -63,6 +63,9 @@ def test_exclude_structured_configs(cfg: Any, ex_false: Any, ex_true: Any) -> No
     ret1 = OmegaConf.to_container(cfg, exclude_structured_configs=False)
     assert ret1 == ex_false
 
+    ret1 = OmegaConf.to_container(cfg, instantiate=True)
+    assert ret1 == ex_true
+
```

Comment:
avoid overloading tests.

You can refactor this to take:
cfg, exclude_structured_configs, expected_result

Also, OmegaConf is overriding __eq__ and return True when a structured config is compared to a matching underlying object.
Your test may be passing because of that. Should probably not use `==` here.
(You can write a more precise eq function for the purpose of this test that is also validating the field type).